haproxy/tests/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "haproxy-tests",
+  "private": true,
+  "version": "1.0.0",
+  "scripts": {
+    "test:errors": "mocha --timeout 10000 errors_generate.spec.js",
+    "test:integration": "mocha --timeout 30000 with_mock.spec.js"
+  },
+  "dependencies": {
+    "chai": "^4.3.7",
+    "chai-exclude": "^2.1.1",
+    "fs-extra": "^11.1.1",
+    "mocha": "^10.2.0",
+    "tmp-promise": "^3.0.3"
```

Comment:
all these dependencies should be installed as dev dependencies in the main package json file.